### PR TITLE
Proposal for feature/issue #8409 - Retry on WebDriver.js acquireSession

### PR DESCRIPTION
**Pull request by elgalu**
This is a working proposal to add retry-on-errors feature to WebDriver.js acquireSession to fix [issue 8409](https://code.google.com/p/selenium/issues/detail?id=8409)

If you are ok with adding this, I'll continue to work on it and, with help, figure out what's missing:
- Figure out how the client will pass the retry `config` through capabilities / other way (easiest would be ENV vars for me) - help appreciated.
- Determine convenient defaults when user pass `config.retry = true` but doesn't specify more details - criteria is appreciated.
- Write unit tests for the feature - help needed!

### CODE REVIEW

**Comment by ValentinH** [comment]
:+1:

File: javascript/webdriver/webdriver.js [hunk 1]
```diff
@@ -148,15 +148,50 @@ webdriver.WebDriver.createSession = function(
...
+      }
+    });
+  }, config.retryTimeout, 'WebDriver acquireSession retry loop.');
+
   return new webdriver.WebDriver(session, executor, flow);
+
+  //-------------------------------------------------------------------------
+
+  // TODO: Improve this poor man sleep sync fn
+  function sleepSync(sleepMs) {
```
Comment (jleyba) [inline javascript/webdriver/webdriver.js]: Use `flow.timeout(sleepMs)`

Comment (elgalu) [inline javascript/webdriver/webdriver.js]: Thanks!

**Comment by jleyba** [comment]
I am not going to accept this change. None of the other language bindings have this functionality and I don't want to set a precedence without a larger discussion on the dev list (which you are welcome to start).

If you are having problems with Chrome crashing, you should file a bug against the [chromedriver](https://code.google.com/p/chromedriver/).

**Pull request closed without merging**